examples/curio/curio-server.py

```diff
@@ -193,13 +193,12 @@ def __init__(self, sock, root):
 if __name__ == '__main__':
     host = sys.argv[2] if len(sys.argv) > 2 else "localhost"
     kernel = Kernel(with_monitor=True)
-    kernel.add_task(h2_server((host, 5000),
-                              sys.argv[1],
-                              "{}.crt.pem".format(host),
-                              "{}.key".format(host)))
     print("Try GETting:")
     print("    On OSX after 'brew install curl --with-c-ares --with-libidn --with-nghttp2 --with-openssl':")
-    print("/usr/local/Cellar/curl/7.47.1/bin/curl --tlsv1.2 --http2 -k https://localhost:5000/bundle.js")
+    print("/usr/local/Cellar/curl/7.50.1/bin/curl --tlsv1.2 --http2 -k https://localhost:5000/bundle.js")
```

Comment:
Can we change this to `/usr/local/opt/curl/bin/curl`? It'll save us from needing to update this regularly.